kbase-extension/scss/partials/_appCell.scss

```diff
@@ -461,3 +445,74 @@ Buttons
   font-size: 75%;
   font-style: italic;
 }
+
+// other bits and pieces
+// spec text display (not used)
+.kb-spec__text {
+  font-size: 80%;
+}
+
+$rcp-height: 50px;
```

Comment:
This covers the 'run control panel', which is the part of the cell with the 'Run' (/Reset/Cancel) button, the tabs, and the cell status line.